### PR TITLE
Update de/en translations

### PR DESCRIPTION
**Pull request by Jan2903**

### CODE REVIEW

**Comment by Tensuko** [comment]
Wow :)
Thanks a lot for the PR!

**Comment by Tensuko** [comment]
Things you have changed in the DE or the EN translation file will be overwritten by the MasterTranslation file.
All changes for DE or EN must be made in the MasterTranslation.

https://github.com/Courseplay/Courseplay_FS22/wiki/Translations
You may do some copy and paste into the right file.

File: config/MasterTranslations.xml [hunk 1]
```diff
@@ -1230,15 +1230,15 @@
...
 		<Translation name="CP_global_setting_stopThreshingDuringRain_title">
-			<Text language="de"><![CDATA[Stoppt Drescher während Regen]]></Text>
-			<Text language="en"><![CDATA[Stops harvester, while it's raining]]></Text>
+			<Text language="de"><![CDATA[Drescher während Regen stoppen]]></Text>
+			<Text language="en"><![CDATA[Stop harvesters while it's raining]]></Text>
 		</Translation>
 		<Translation name="CP_global_setting_stopThreshingDuringRain_tooltip">
 			<Text language="de"><![CDATA[Wenn die Einstellung aktiviert ist, dann warten alle Drescher solange, bis der Regen aufhört.]]></Text>
-			<Text language="en"><![CDATA[If the setting is active, then all harvester wait for the raining to stop.]]></Text>
+			<Text language="en"><![CDATA[If the setting is active, then all harvesters wait for the raining to stop.]]></Text>
```
Comment (pvaiko) [inline config/MasterTranslations.xml:1238]: If the setting is active, all harvesters will wait for the rain to stop.

File: config/MasterTranslations.xml [hunk 2]
```diff
@@ -2957,27 +2967,27 @@ Der Versatz der Schaufelhöhe dient dazu, die Höhe über dem Boden anzupassen.
...
+Schaufeln mit Gebiss öffnen dieses automatisch beim Laden und Abladen und schließen es zum Transport.
 Die Schaufel zum Zerkleinern von Zuckerrüben wird mit allen Funktionen unterstützt.
 ]]></Text>
 			<Text language="en"><![CDATA[
 This mode enables wheel loader and tractors with a front loader. 
 With a shovel attached a heap can be picked up from the ground or loading from a bunker silo can be accomplished.
 The picked up fill level can be unloaded automatically into a nearby trailer or a selected unloading station like a BGA.
 The shovel will automatically be moved to the correct positions for loading, unloading and so on.
-In case the mode is not directly visable on the hud, you can toggle the "start position" until the shovel mode appears.
+In case the mode is not directly visable on the HUD, you can toggle the "start position" until the shovel mode appears.
```
Comment (pvaiko) [inline config/MasterTranslations.xml:2978]: visible

**Review by pvaiko — CHANGES_REQUESTED** [review]

**Pull request merged**